android/app/src/main/res/layout/item_track.xml

```diff
@@ -1,43 +1,51 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Removed parent LinearLayout. Sorry for comitting changes in gradle.properties again and again, my college internet works on proxy so they are added automatically. I restrored it to its original state.